sensor_msgs/Image rgb_img
---
fcsc/ObjectMaskArray masks
